src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -1012,4 +986,34 @@ - (nullable NSData *)fetchAttestationChallengeForDeviceId:(uint64_t)deviceId
     return [self fetchAttestationChallengeForDeviceID:deviceId];
 }
 
+- (BOOL)getBaseDevice:(uint64_t)deviceID queue:(dispatch_queue_t)queue completionHandler:(MTRDeviceConnectionCallback)completion
+{
```

Comment:
Nit, this should be dispatching to the new rename and have the body there.

Reply:
Actually, this is purposeful.  The new rename is going to go away as I keep cherry-picking in the API change PRs, so doing it that way would actually make the changes larger going forward.  The end state will be that the new rename does not exist and this backwards-compat shim keeps doing what it's doing now...